js/localization.js

```diff
@@ -103,7 +103,11 @@ function tNodeList(nodes) {
 
 function tDocLoader() {
   t.DOMParser = new DOMParser();
-  t.cache = tryJSONparse(localStorage.L10N) || {};
+  t.cache = (function () {
+    try {
+      return JSON.parse(localStorage.L10N);
+    } catch (e) {}
+  })() || {};
 
```

Comment:
Why duplicate the `tryJSONparse` function (in `messaging.js`)?

Reply:
to remove the dependency so that localization.js can be loaded separately